<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sessionTab</class>
 <widget class="QWidget" name="sessionTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_misc">
     <item>
      <widget class="QLabel" name="label_friendName">
       <property name="text">
        <string>foo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_connectStatus">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea_Sessioncontent">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>613</width>
        <height>415</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_send">
     <item>
      <widget class="QLineEdit" name="lineEdit_textToSend"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_send">
       <property name="text">
        <string>SEND</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
